lib/version.rb

```diff
@@ -1,3 +1,4 @@
 module DangerSwiftlint
-  VERSION = "0.3.0".freeze
+  VERSION = "0.16.1".freeze
```

Comment:
Yes it is, my goal to mirror both versions I believe is two fold 1) It is easy/quick for us to know which version of swiftlint will be used and what it is capable of and how much we need to "workaround" any limitation it has 2) From the user perspective they will be able to choose which version makes sense for them, perhaps they are not ready for a upgrade or they want to ensure always will use a given swiftlint version, etc, leading to a more predictable less flaky build.